ref/feature/jaxrsClient/examples.adoc

```diff
@@ -0,0 +1,51 @@
+== Examples
+
+**LC: Could do with an introductory sentence here...**
+
+=== Overriding JAX-RS Client configuration when moving to production
+The JAX-RS Client APIs allow you to specify properties on the client that configure settings like timeouts, proxy
+hosts, SSL properties, and so on. If the application is deployed in a different environment, these settings might need to change. For example, an application might initially be written to run in a development environment, where all remote RESTful services are available without a proxy server and can connect very quickly:
+
+[source,java]
+----
+public Widget getRemoteWidget(String id) {
+    Client client = ClientBuilder.newClient();
+    WebTarget target = client.target("https://somehost:9443/WidgetStore/widgets/");
+    target.property("com.ibm.ws.jaxrs.client.connection.timeout", 5000); // 5 seconds
+    target.property("myCustomProperty", "dev");
+    Widget widget = target.path(id).request().get(Widget.class);
+    return widget;
+}
+----
+
+When this application is deployed in a production environment, however, it might be necessary to modify these settings. This can be accomplished by adding the following configuration to the server config (`server.xml`):
+
+**LC: I think this implies that you can override the Java values with the server.xml values when that isn't what happens. Can you change how this is explained? I assume you need to leave the values out of the Java code for this to work?**
+
+[source,xml]
+----
+<webTarget uri="https://somehost:9443/WidgetStore*" proxyHost="myProxyHost" proxyPort="55555" proxyType="HTTP"
+           connectionTimeout="30000" receiveTimeout="30000" />
+----
+
+This changed configuration alters the `WebTarget` in the Java code to use the specified HTTP proxy server (`myProxyHost:55555`) and use connection and receive timeouts of 30 seconds (30,000 milliseconds).
```

Comment:
```suggestion
This configuration alters the `WebTarget` in the Java code to use the specified HTTP proxy server (`myProxyHost:55555`) and use connection and receive timeouts of 60 seconds (60,000 milliseconds) instead of the default timeouts.
```